[2022-12-04 11:00:02,806] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:30:00+00:00 [queued]>
[2022-12-04 11:00:02,864] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:30:00+00:00 [queued]>
[2022-12-04 11:00:02,868] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:00:02,871] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-04 11:00:02,872] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:00:02,935] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-11-03 13:30:00+00:00
[2022-12-04 11:00:02,943] {standard_task_runner.py:52} INFO - Started process 11378 to run task
[2022-12-04 11:00:02,974] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'local_to_gcs_task', 'scheduled__2022-11-03T13:30:00+00:00', '--job-id', '4878', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmpy27xh9f6', '--error-file', '/tmp/tmpkuihlucl']
[2022-12-04 11:00:02,982] {standard_task_runner.py:77} INFO - Job 4878: Subtask local_to_gcs_task
[2022-12-04 11:00:03,182] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:30:00+00:00 [running]> on host 9e44869ff3ef
[2022-12-04 11:00:03,682] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-03T13:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-03T13:30:00+00:00
[2022-12-04 11:00:04,505] {python.py:175} INFO - Done. Returned value was: None
[2022-12-04 11:00:04,665] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=local_to_gcs_task, execution_date=20221103T133000, start_date=20221204T110002, end_date=20221204T110004
[2022-12-04 11:00:04,951] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 11:00:05,528] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
